src/transformers/models/auto/configuration_auto.py

```diff
@@ -175,6 +175,8 @@
         ("transfo-xl", "TransfoXLConfig"),
         ("trocr", "TrOCRConfig"),
         ("tvlt", "TvltConfig"),
+        # ("udop_dual", "UdopConfig"),
```

Comment:
@NielsRogge I am not sure how to handle here. There are two models in UDOP, both share same config. 

How do I setup MODEL_FOR_PRETRAINING_MAPPING_NAMES ? 
How do I setup CONFIG_MAPPING_NAMES ?

Reply:
This just needs to be `("udop", "UdopConfig")`. "udop" is the model_type.